it.js

```diff
@@ -581,6 +585,7 @@ export const cuss = {
   more: 0,
   mori: 0,
   moro: 0,
+  muori: 2,
```

Comment:
And this is `die` right? That seems like it might also sometimes be used in normal, clean text?